### PR TITLE
fix: improvements to Dockerfiles

### PR DESCRIPTION
**Pull request by tricky42**
## Proposed changes
- expose ory home as volume to simplify passing in own config file
- declare Kratos default ports in Dockerfile

## Checklist
- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: .docker/Dockerfile-build [hunk 1]
```diff
@@ -32,6 +30,13 @@ RUN mkdir -p /var/lib/sqlite
 RUN chown ory:ory /var/lib/sqlite
 VOLUME /var/lib/sqlite
 
+# Exposing the ory home directory to simplify passing in Kratos configuration (e.g. if the file $HOME/.kratos.yaml 
+# exists, it will be automatically used as the configuration file).
+VOLUME /home/ory
```
Comment (aeneasr) [inline .docker/Dockerfile-build:35]: Would it make sense to document that somewhere?

Comment (tricky42) [reply .docker/Dockerfile-build:35]: Yes, definitely. I first wanted to get the blessing on the general changes. I think it makes sense to add a section on the Dockerfiles and how to work with them. 

The documentation should cover: 
* different types of images (with / without SQLite)
* configuration management (mout config file, custom image)
* inject "secrets" via ENV)

Comment (aeneasr) [reply .docker/Dockerfile-build:35]: Sounds good, this is good to merge from my side - should I go ahead?

Comment (tricky42) [reply .docker/Dockerfile-build:35]: Yes. So Documentation will follow in another PR, ok?

Comment (aeneasr) [reply .docker/Dockerfile-build:35]: yup!

**Pull request merged**